README.md

```diff
@@ -49,11 +49,8 @@ Another alternative is to run it as a container, but this will require to mount
 the docker socket inside the container:
 
 ```sh
-docker build . -t aojea/cloud-provider-kind:v0.1
-# using the host network
-docker run --rm --network host -v /var/run/docker.sock:/var/run/docker.sock aojea/cloud-provider-kind:v0.1
-# or the kind network
-docker run --rm --network kind  -v /var/run/docker.sock:/var/run/docker.sock aojea/cloud-provider-kind:v0.1
+# using the `kind` network (`host` is the default value for NET_MODE)
+NET_MODE=kind docker compose up -d
```

Comment:
I feel like we should document how to do this without compose still.

E.G. if we start to publish a container image that is pre-built (when the project is more mature) then you will want to skip building most of the time and we'll want this simple docker run step again (or binary run on the host)

We're also going to want to figure out e.g. podman, where you won't be mounting the socket necessarily.